src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -65,29 +65,60 @@ private static bool UseLoggerMessageDefine(LoggerMethod lm)
 
             private void GenType(LoggerClass lc)
             {
+                int indentationSize = 0;
```

Comment:
Why do we need to keep track of both `int indentationSize` and `string nestedIndentation`? Couldn't we make due with just the string, and append / trim 4 spaces as necessary?

Reply:
removed `indentationSize`